model/status_diff.go

```diff
@@ -161,5 +169,35 @@ func StatusDiffTasks(original *task.Task, patch *task.Task) TaskStatusDiff {
 				Patch:    getTestUrl(&test),
 			})
 	}
+
+	return diff
+}
+
+// StatusDiffTests takes two sets of tests and returns a diff of their results
+// for easy comparison and analysis.
+func StatusDiffTests(original, patch []task.TestResult) []TestStatusDiff {
+	diff := []TestStatusDiff{}
+	if len(original) == 0 || len(patch) == 0 {
+		return diff
+	}
+
+	originalMap := make(map[string]task.TestResult)
+	for _, test := range original {
+		originalMap[test.TestName()] = test
+	}
+
+	for _, test := range patch {
+		baseTest := originalMap[test.TestName()]
+
+		// Get the base name for windows/non-windows paths.
+		testName := path.Base(strings.Replace(test.TestName(), "\\", "/", -1))
```

Comment:
Is this supposed to be `filepath.Base`? `path.Base` is for URLs.

Alternatively, you could just use `util.ConsistentFilepath`.

Reply:
I copy/pasted this from StatusDiffTasks. It works and this is going to be gone once we completely switch to the spruce UI.